### ISSUE TITLE
Devuan

### ISSUE THREAD

**Issue by Minda1975** [dylanaraps/pfetch#163]
Hello.  Thanks for this little project. I have a small request. Would it be possible to include the Devuan logo in this project?

**Comment by dylanaraps** [comment]
Just missing ASCII art.

**Comment by Minda1975** [comment]
> Just missing ASCII art.

Yes.  Pfetch shows a generic penguin logo. It would certainly be good if this program could display Devuan logo.
